src/assets/messages.json

```diff
@@ -3351,7 +3351,12 @@
     "block-config-accuracy_timeout_seconds-type": "Accuracy timeout in seconds should be a number",
     "block-config-value-type": "Output expression should be a string",
     "block-config-is_member-type": "Value/Expression should be a string",
-    "resource-values-x-value-pattern": "Content should only contain basic Latin script, digits, or punctuation"
+    "resource-values-x-value-pattern": "Content should only contain basic Latin script, digits, or punctuation",
+    "numeric-block-min-value-must-be-lower-than-max-value": "Minimum value (inclusive) must be lower than Maximum value (inclusive)",
+    "numeric-block-max-value-must-be-greater-than-min-value": "Maximum value (inclusive) must be greatter than Minimum value (inclusive)",
```

Comment:
```suggestion
    "numeric-block-max-value-must-be-greater-than-min-value": "Maximum value (inclusive) must be greater than Minimum value (inclusive)",
```

Reply:
Perhaps, _Maximum value must be greater or equal to Minimum value_ would be easier to understand?

Reply:
@object-required , thanks for suggesting an improvement here. I added the `or equal` for this, and for the min-value above ! But for the rest, I'd like to leave it for designers, they will still revisit all those texts.